.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 -   repo: https://github.com/PyCQA/isort.git
-    rev: 5.8.0
+    rev: 5.12.0
```

Comment:
I think the docs also need some update no?

Reply:
Clarified - no need to specify the pre-commit hook versions in the docs :)